remote_config/CMakeLists.txt

```diff
@@ -45,15 +45,15 @@ build_flatbuffers("${flatbuffer_schemas}"
 # Common source files used by all platforms
 set(common_SRCS
     src/common.cc
-    src/remote_config.cc
-    src/config_update_listener_registration.cc)
+    src/remote_config.cc)
 
 # Source files used by the Android implementation.
 set(android_SRCS
     src/android/remote_config_android.cc)
 
 # Source files used by the iOS implementation.
 set(ios_SRCS
+    src/ios/config_update_listener_registration.cc
```

Comment:
Is this file also going to be used by android? If so it should stay under the common source

Reply:
This implementation was iOS specific. I'm not sure how Android will implement the registration.

Reply:
I'll either make an android version of this file or move it back to common if it generalizes well enough.